pkg/minikube/out/out.go

```diff
@@ -191,6 +191,17 @@ func SetOutFile(w fdWriter) {
 func SetJSON(j bool) {
 	klog.Infof("Setting JSON to %v", j)
 	JSON = j
+	if JSON {
+		if err := flag.Set("logtostderr", "false"); err != nil {
```

Comment:
In Go, libraries should never set or read flags - only `main`.

Reply:
thanks for this comment - i've removed it altogether (as per your general comment), and this specifically is because libraries should never (unexpectedly and uncontrollably) change the behaviour of the mail app?